src/styles/version-switcher.scss

```diff
@@ -0,0 +1,38 @@
+.version-switcher {
+    gap: 0;
+
+    &__link {
+        flex: 1 1 0;
+        overflow: hidden;
+        text-decoration: none;
+
+        &:first-of-type {
+            border-top-left-radius: var(--amplify-radii-medium);
+            border-bottom-left-radius: var(--amplify-radii-medium);
+        }
+
+        &:last-of-type {
+            border-top-right-radius: var(--amplify-radii-medium);
+            border-bottom-right-radius: var(--amplify-radii-medium);
+        }
+    }
+
+    &__wrapper {
+        height: 100%;
+        background-color: var(--amplify-colors-overlay-5);
+        padding: var(--amplify-space-small);
+        white-space: nowrap;
+        padding: var(--amplify-space-xs);
+        text-align: center;
+
+        &.active {
+            background-color: var(--amplify-colors-teal-40);
```

Comment:
The contrast for this is about 3.4:1. Can we keep this background color but bump the text color to `var(--amplify-colors-font-primary)`? That will give us the same contrast ratio as the primary button color.